jscomp/frontend/ppx_entry.ml

```diff
@@ -30,6 +30,7 @@ let rewrite_signature (ast : Parsetree.signature) : Parsetree.signature =
   let ast =
     match !Js_config.jsx_version with
     | 3 -> Reactjs_jsx_ppx_v3.rewrite_signature ast
+    | 4 -> Reactjs_jsx_ppx_v4.rewrite_signature !Js_config.jsx_mode ast
```

Comment:
Style: perhaps use a labeled argument `~jsx_mode` here as first argument to `Reactjs_jsx_ppx_v4.rewrite_signature`.

Reply:
Got it!